src/__tests__/Button.test.tsx

```diff
@@ -91,8 +91,9 @@ describe('Button', () => {
   })
 
   it('styles icon only button to make it a square', async () => {
-    const container = render(<IconButton icon={SearchIcon} iconLabel="Search icon only button" />)
+    const container = render(<IconButton icon={SearchIcon} aria-label="Search button" />)
```

Comment:
I think we should have a separate test for the aria-label / aria-labelledby. thoughts?

also a suggestion for your matcher, to be more specific around looking for the aria-label attributes:

```diff
- findByRole('button')
+ getByRole('button', {name: 'Search button'} ) 
```

I don't think you need use an async matcher? 🤔